src/jit/SimdArm64Inl.h

```diff
@@ -413,6 +445,26 @@ static void emitBinarySIMD(sljit_compiler* compiler, Instruction* instr)
     case ByteCode::I16X8GeUOpcode:
         simdEmitOp(compiler, SimdOp::ucmge | SimdOp::H8, dst, args[0].arg, args[1].arg);
         break;
+    case ByteCode::I16X8ExtmulLowI8X16SOpcode:
+        simdEmitOp(compiler, SimdOp::sxtl | (0x1 << 19), args[0].arg, args[0].arg, 0);
```

Comment:
Is this change args[0]?

Reply:
yes

Reply:
It could be a problem when register allocation is implemented. It should use dst and maybe a tmp register instead.